models/spot.js

```diff
@@ -65,7 +102,8 @@ exports.updateRating = function(spotId, rating) {
   return firebaseRef.child(spotId).transaction(function(currentSpot) {
     // Data can be null here because of the way transactions sync w/ local data.
     if (currentSpot === null) {
-      return {rating: rating, numRatings: 1};
+      // weird workaround to give transaction() a default local value
+      return new Spot(0, 0, 0, 0, 0, 0, 0, 0).attributes;
```

Comment:
FB needs to have some valid default when we first poll our local copy. This won't ever be written, but it'll check if this diffs from the server. If so, it reruns this function(currentSpot) where currentSpot = the actual server value.